src/main/java/cubicchunks/server/ServerCubeCache.java

```diff
@@ -131,12 +131,14 @@ public Column loadChunk(int columnX, int columnZ) {
 	@Override
 	@Nullable
 	public Column loadChunk(int columnX, int columnZ, Runnable runnable) {
-		Column column = this.getColumn(columnX, columnZ, /*Requirement.LOAD*/Requirement.LIGHT);
-		if (runnable == null) {                          // TODO: Set this to LOAD when PlayerCubeMap works
-			return column;
+		// TODO: Set this to LOAD when PlayerCubeMap works
+		if (runnable == null) {
+			return getColumn(columnX, columnZ, /*Requirement.LOAD*/Requirement.LIGHT);
```

Comment:
Requirement `LIGHT` doesn't seem to make sense for column. Why do you even use cube requirements for columns?